### PR TITLE
[AWS] [Lambda] Spec

### PR DESCRIPTION
**Pull request by nagypeterjob**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**

**What is the current behavior? (You can also link to an open issue here)**

**What is the new behavior (if this is a feature change)?**

**Additional info:**

### CODE REVIEW

File: dev/aws/lambda.ts [hunk 1]
```diff
@@ -1,3 +1,1001 @@
...
+    cache: {
+      ttl: ttl,
+    },
+  },
+
+  listDestinationConfigArns: {
+    custom: async function (context, executeShellCommand) {
+      // Getting sqs queues is implemented, altough it has a huge performance toll.
+      // It seems Fig rejects long-running promises after a time.
+      // I am currently investigating if this is the case.
```
Comment (QuiiBz) [inline dev/aws/lambda.ts:747]: Fig as a default timeout of 5 seconds for generators. Does `aws sqs list-queues` takes longer than that?

Comment (nagypeterjob) [reply dev/aws/lambda.ts:747]: Yes, we would need more then 5 sec here

Comment (nagypeterjob) [reply dev/aws/lambda.ts:747]: but this is not a blocker

Comment (QuiiBz) [reply dev/aws/lambda.ts:747]: Let's follow #270, #96 to get this implemented later

**Review by QuiiBz — APPROVED** [review]

**Pull request merged**